debezium-core/src/main/java/io/debezium/relational/ddl/AbstractDdlParser.java

```diff
@@ -0,0 +1,365 @@
+        signalEvent(new DdlParserListener.TableDroppedEvent(id, statement, true));
+    }
+
+    /**
+     * Signal a create index event to all listeners.
+     *
+     * @param indexName the name of the index; may not be null
+     * @param id        the table identifier; may be null if the index does not apply to a single table
+     * @param statement the DDL statement; may not be null
+     */
+    protected void signalCreateIndex(String indexName, TableId id, String statement) {
+        signalEvent(new DdlParserListener.TableIndexCreatedEvent(indexName, id, statement));
+    }
+
+    /**
+     * Signal a drop index event to all listeners.
+     *
+     * @param indexName the name of the index; may not be null
+     * @param id        the table identifier; may not be null
+     * @param statement the DDL statement; may not be null
+     */
+    protected void signalDropIndex(String indexName, TableId id, String statement) {
+        signalEvent(new DdlParserListener.TableIndexDroppedEvent(indexName, id, statement));
+    }
+
+    protected String removeLineFeeds(String input) {
+        return input.replaceAll("[\\n|\\t]", "");
+    }
+
+    /**
+     * Utility method to accumulate a parsing exception.
+     *
+     * @param e    the parsing exception
+     * @param list the list of previous parsing exceptions; may be null
+     * @return the list of previous and current parsing exceptions; if {@code e} is null then always {@code list}, but otherwise non-null list
+     */
+    protected Collection<ParsingException> accumulateParsingFailure(ParsingException e, Collection<ParsingException> list) {
+        if (e == null) return list;
+        if (list == null) list = new ArrayList<ParsingException>();
+        list.add(e);
+        return list;
+    }
+
+    /**
+     * Utility method to accumulate a parsing exception.
+     *
+     * @param e    the multiple parsing exceptions
+     * @param list the list of previous parsing exceptions; may be null
+     * @return the list of previous and current parsing exceptions; if {@code e} is null then always {@code list}, but otherwise non-null list
+     */
+    protected Collection<ParsingException> accumulateParsingFailure(MultipleParsingExceptions e, Collection<ParsingException> list) {
+        if (e == null) return list;
+        if (list == null) list = new ArrayList<ParsingException>();
+        list.addAll(e.getErrors());
+        return list;
+    }
+
+    protected Column createColumnFromConstant(String columnName, String constantValue) {
+        ColumnEditor column = Column.editor().name(columnName);
+        try {
+            if (constantValue.startsWith("'") || constantValue.startsWith("\"")) {
+                column.type("CHAR");
+                column.jdbcType(Types.CHAR);
+                column.length(constantValue.length() - 2);
+            } else if (constantValue.equalsIgnoreCase("TRUE") || constantValue.equalsIgnoreCase("FALSE")) {
```

Comment:
Please let's have no code on a line after a closing "}". I know it's currently a bit inconsistent in the code base, but we recently agreed with @jpechane on only using this style in new code going forward.

Reply:
I have never seen this code style in Java. I will update my code style rules for code formatting and try to reformat code created by me

Reply:
rev. 970d16bf6667ee6267153608937a01e15c37efb1